cataclysmbnteam/Cataclysm-BN
# Simplify broken limb mending

#### Summary



SUMMARY: Balance "Simplify broken limb mending"

#### Purpose of change



Make broken limb not take you out of combat for [insert way too long duration here].

#### Describe the solution



Broken limbs can now be healed in every way a healthy limb can, they just heal at 25% speed, unless splinted or you have REGEN_LIZ.
Limbs stay broken until they reach full hp.

#### Describe alternatives you've considered



* Reworking limb breaking to be purely hp based, so that limb is broken when below x% hp
* Keeping the long wait until we have proper limb removal

#### Testing



* Set hp of limbs to, respectively: 0, 0, 0, 50%
* Set one of the 0 hp limbs to 50% hp
* Splint one of the 0 hp limbs
* The splinted limb should now be "blue" on the hp bar, two broken ones should be red
* Sleep until the splinted limb visibly regens about 25%
* Splinted limb should regen as much as the non-broken limb
* The non-splinted limbs should heal only about 1/4 of that amount
* Set hp of all broken limbs to full
* Note they unbreak on the next turn
* Set leg hp to 0, 0, then up to 50%, 50%, then wait a turn
* Try to run - should fail
* Set leg hp to 100%, 100%, then wait a turn
* Try to run - should succeed
* Set arm hp to 0, 100%, then 50%, 100%
* Try to fire a bow - should fail
* Set arm hp to 100%, 100%, then wait a turn
* Try to fire a bow - should succeed

#### Additional context
